core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the stories list.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { Subtopic } from 'domain/topic/SubtopicObjectFactory';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { PracticeSessionPageConstants} from
+  'pages/practice-session-page/practice-session-page.constants.ts';
+
+@Component({
+  selector: 'practice-tab',
+  templateUrl: './practice-tab.component.html',
+  styleUrls: []
+})
+export class PracticeTabComponent implements OnInit {
+  @Input() topicName: string;
+  @Input() subtopicsList: Array<Subtopic>;
+  selectedSubtopics: Array<Subtopic> = [];
+  availableSubtopics: Array<Subtopic> = [];
+  selectedSubtopicIndices: Array<Boolean> = [];
+  constructor(
+    private urlInterpolationService: UrlInterpolationService
+  ) {}
+  ngOnInit(): void {
```

Comment:
Remove void.

Reply:
#9759 (comment) Was just using this as reference